src/layers/layers.component.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  Component,
+  QueryList,
+  HostBinding,
+  ContentChildren,
+  AfterContentChecked,
+  ChangeDetectionStrategy, Input
+} from '@angular/core';
+import { DomSanitizer, SafeHtml } from '@angular/platform-browser';
+import { layer, Layer, Text, Icon } from '@fortawesome/fontawesome';
+import { compact } from 'lodash';
```

Comment:
Feels like an overkill to pull in whole lodash library for a aingle simple function. Can it be inlined?

Reply:
I wondered about that too. But wouldn't this be tree-shaken in the build? i.e. inlined? That's what Rollup would do with that named import.

Reply:
You're right, it can be implemented with simple filter method. I'll change and commit this soon.
Just for saying, it was a mistake. At the beginning I used additional lodash methods but after refactoring I probably removed them all, except compact whose slipped out :)